modules/network-observability-netobserv-cli-install.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+
+// * observability/network_observability/network-observability-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-cli-install_{context}"]
+= Installing the netobserv CLI
+Installing the `netobserv` CLI is a separate procedure from the Operator installation. This means that, even if you have the Network Observability Operator installed from the Operator Hub, you need to install the CLI separately. 
+
+.Prerequisites
+* Install the OpenShift CLI (oc)
+
+.Procedure
+//replace the link with non-github replacement when Julien figures out where to put it.
+. Navigate to the link:https://github.com/netobserv/network-observability-cli/releases/download/v0.0.1/netobserv-cli.tar.gz[netobserv cli tar file].
+. Unpack the archive:
++
+[source,terminal]
+----
+$ tar xvf netobserv-cli.tar.gz
+----
+. Make the file executable:
++
+[source,terminal]
+----
+$ chmod +x netobserv-cli
+----
+. Move the extracted `netobserv-cli` binary to a directory that is on your `PATH`:
++
+[source,terminal]
+----
+$ sudo mv ./netobserv-cli /usr/local/bin/netobserv-cli
```

Comment:
Currently, the tar archive contains a `/build/` folder containing `netobserv` script.
When CPaaS will build it, it should contains a `oc-netobserv` script.

The tar also contains a LISENCE at its root. We may add a README on top.

The commands should be:
```suggestion
. Make the file executable:
+
[source,terminal]
----
$ chmod +x ./build/oc-netobserv
----
. Move the extracted `netobserv-cli` binary to a directory that is on your `PATH`:
+
[source,terminal]
----
$ sudo mv ./build/oc-netobserv /usr/local/bin/.
```

Reply:
Okay got it, I will go ahead and update these then. Does the `.` need to be at the end of that `sudo mv...`?